.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
```

Comment:
> stage-0

What ES7 features do you want to use? I think it's better to name those and include those.